api/src/opentrons/protocol_engine/errors/exceptions.py

```diff
@@ -47,6 +47,10 @@ class LabwareNotLoadedError(ProtocolEngineError):
     """An error raised when referencing a labware that has not been loaded."""
 
 
+class LabwareNotOnDeckError(ProtocolEngineError):
+    """An error raised when referencing a labware that was moved off-deck."""
```

Comment:
Would this be clearer?

```suggestion
    """An error raised when a labware can't be used because it's off-deck."""
```

Because *some* kinds of references to off-deck labware are okay. For example, when you're trying to move an off-deck labware back on-deck.

No strong opinion from me either way.